WordPress/Classes/ViewRelated/Reader/Controllers/ReaderStreamViewController.swift

```diff
@@ -214,11 +214,6 @@ import AutomatticTracks
     /// - Returns: An instance of the controller
     ///
     @objc class func controllerWithTopic(_ topic: ReaderAbstractTopic) -> ReaderStreamViewController {
-        // if a default discover topic is provided, treat it as a site to retrieve the header
-        if ReaderHelpers.topicIsDiscover(topic) {
```

Comment:
I don't think we need this anymore. The worst case scenario, it'll just open as a regular site feed.